conversations/v3/whatsapp.yaml

```diff
@@ -549,7 +549,7 @@ paths:
         - "Phone Number Configurations"
       summary: List Configurations
       operationId: listAllWhatsAppChannels
-      description: List all WhatsApp channels available to your API account
+      description: List all WhatsApp phone number configurations available to your API account
```

Comment:
Here I suggest whole section. Using WhatsApp phone number itself is too confusing.

Reply:
@ridlees, ummm... Actually... This has to be understand as a "phone number configuration" for WhatsApp (channel)... If you still think that it should be "WhatsApp Business API phone number configuration" (which sounds pretty long, BTW), what would you suggest to do with the "phone number configuration" section?

Reply:
Let's discard it then and I will raise the naming concern somewhere else at different time. (I see your point, but that doesn't mean I like how it looks since it adds to existing confusion).

Reply:
Maybe you want to raise the concern here: https://github.com/tyntec/api-collection/pull/102/files#r827380082 ?